apps/native-component-list/src/screens/WebBrowser/WebBrowserScreen.tsx

```diff
@@ -0,0 +1,146 @@
+export default class WebBrowserScreen extends React.Component<{}, State> {
+  static navigationOptions = {
+    title: 'WebBrowser',
+  };
+
+  readonly state: State = {};
+
+  componentDidMount() {
+    if (Platform.OS === 'android') {
+      WebBrowser.getCustomTabsSupportingBrowsersAsync()
+        .then(({ browserPackages }) =>
+          browserPackages.map((name) => ({ label: name, value: name }))
+        )
+        .then((packages) => this.setState({ packages }));
+    }
+  }
+
+  componentWillUnmount() {
+    if (Platform.OS === 'android') {
+      WebBrowser.coolDownAsync(this.state.lastWarmedPackage);
+    }
+  }
+
+  showPackagesAlert = async () => {
+    const result = await WebBrowser.getCustomTabsSupportingBrowsersAsync();
+    Alert.alert('Result', JSON.stringify(result, null, 2));
+  };
+
+  handleWarmUpClicked = async () => {
+    const { selectedPackage: lastWarmedPackage } = this.state;
+    this.setState({
+      lastWarmedPackage,
+    });
+    const result = await WebBrowser.warmUpAsync(lastWarmedPackage);
+    Alert.alert('Result', JSON.stringify(result, null, 2));
+  };
+
+  handleMayInitWithUrlClicked = async () => {
+    const { selectedPackage: lastWarmedPackage } = this.state;
+    this.setState({
+      lastWarmedPackage,
+    });
+    const result = await WebBrowser.mayInitWithUrlAsync(url, lastWarmedPackage);
+    Alert.alert('Result', JSON.stringify(result, null, 2));
+  };
+
+  handleCoolDownClicked = async () => {
+    const result = await WebBrowser.coolDownAsync(this.state.selectedPackage);
+    Alert.alert('Result', JSON.stringify(result, null, 2));
+  };
+
+  packageSelected = (value: string | number) => {
+    if (typeof value === 'string') {
+      this.setState({ selectedPackage: value });
+    }
+  };
+
+  renderAndroidChoices = () =>
+    Platform.OS === 'android' && (
+      <>
+        <View style={styles.label}>
+          <Text>Force package:</Text>
+          <Picker
+            style={styles.picker}
+            selectedValue={this.state.selectedPackage}
+            onValueChange={this.packageSelected}>
+            {this.state.packages &&
+              [{ label: '(none)', value: '' }, ...this.state.packages].map(({ value, label }) => (
+                <Picker.Item key={value} label={label} value={value} />
+              ))}
+          </Picker>
+        </View>
+      </>
+    );
+
+  renderAndroidButtons = () =>
+    Platform.OS === 'android' && (
+      <>
+        <Button style={styles.button} onPress={this.handleWarmUpClicked} title="Warm up." />
+        <Button
+          style={styles.button}
+          onPress={this.handleMayInitWithUrlClicked}
+          title="May init with url."
+        />
+        <Button style={styles.button} onPress={this.handleCoolDownClicked} title="Cool down." />
+        <Button
+          style={styles.button}
+          onPress={this.showPackagesAlert}
+          title="Show supporting browsers."
+        />
```

Comment:
```suggestion
        [
          {
            onPress: this.handleWarmUpClicked,
            title: "Warm up.",
          },
          {
            onPress: this.handleMayInitWithUrlClicked,
            title: "May init with url.",
          },
          {
            onPress: this.handleCoolDownClicked,
            title: "Cool down.",
          },
          {
            onPress: this.showPackagesAlert,
            title: "Show supporting browsers.",
          },     
        ].map(({ title, onPress }, idx) => (
          <Button style={styles.button} key={idx} onPress={onPress} title={title} />
        ))
```

Reply:
I just left this unchanged because it's something that is Android-specific and so far I was focused only on iOS.